crates/flipperzero/src/furi/mod.rs

```diff
@@ -4,6 +4,8 @@ pub mod io;
 pub mod log;
 pub mod message_queue;
 pub mod rng;
+#[cfg(feature = "alloc")]
+pub mod stream_buffer;
```

Comment:
It feels really bloated to a feature just because of the `Arc` I used. I could replace it with some custom counting logic using an `AtomicUsize` and `sys::malloc` to get a pointer but I'm not sure if that is better.